widgets/GCSBucketCreate-action.json

```diff
@@ -56,6 +56,19 @@
           }
         }
       ]
+    },
+    {
+      "label": "Auto Create",
+      "properties": [
+        {
+          "widget-type": "textbox",
```

Comment:
can it be a selector?

Reply:
This is very complicated since they can be single region or multi region, it is difficult to list all of them.

Reply:
selector is brittle, it means we have to update CDAP in order to accommodate a new location. Unless selector has a way to overwrite with a custom value, I think this should be a textbox